release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -243,9 +243,9 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 
 [discrete]
 [id="ocp-4-16-sha-haproxy-support-removed"]
-=== SHA1 certificates no longer supported for use with HAProxy
+=== SHA-1 certificates no longer supported for use with HAProxy
 
-SHA1 certificates are no longer supported for use with HaProxy. Both existing and new routes using SHA1 certificates in {product-version} are rejected and no longer function.
+SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Comment:
I might leave the {product-title} in there to be clear. I like the addition of the version number.
```suggestion
SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in {product-title} 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Reply:
Sure thing!